site/zenodo_rdm/migrator/transform/records/records.py

```diff
@@ -83,13 +83,12 @@ def _custom_fields(self, entry):
 class ZenodoRecordTransform(RDMRecordTransform):
     """Zenodo to RDM Record class for data transformation."""
 
-    def _community_id(self, entry):
+    def _communities(self, entry):
         communities = entry["json"].get("communities")
         if communities:
+            slugs = [slug for slug in communities]
             # TODO: handle all slugs
-            slug = communities[0]
-            if slug:
-                return {"ids": [slug], "default": slug}
+            return {"ids": slugs, "default": slugs[0]}  # TODO default?
```

Comment:
To be discussed: should the default community be the first one from the list of the record's communities? The notion of `default` did not exist on zenodo so we need to decide what do we want as default. 

I can't remember of better suggestions except doing something like "prioritizing". E.g: prioritize some communities like Biodiversity Literature Repository, LORY, etc. If not found, default to the first one from the list.

Reply:
Good tip, thanks!

https://github.com/zenodo/zenodo-rdm/issues/259